stories/generictile/generic-tile.css

```diff
@@ -1,3 +0,0 @@
-.fd-tile {
```

Comment:
We should never be overriding any `fd-*` classes for documentation. Add a custom `fddocs-` class instead.